.github/workflows/ci.yml

```diff
@@ -27,87 +31,69 @@ jobs:
         DATABASE_PORT: 3306
         SECRET_KEY: ${{ secrets.SECRET_KEY }}
         DJANGO_SETTINGS_MODULE: answerking.settings.development
-
+    
     steps:
-    - uses: actions/checkout@v3
-
-    - name: Install poetry
-      run: pipx install poetry
-
-    - name: Set up Python 3.10
-      uses: actions/setup-python@v4
-      with:
-        python-version: '3.10'
-        cache: 'poetry'
-
-    - run: poetry install
-
-    - name: Run Migrations
-      run: |
-        poetry run python manage.py makemigrations
-        poetry run python manage.py migrate
-
-    - name: Run Unit Tests
-      run: poetry run python manage.py test
+      - uses : actions/checkout@v3
+      - uses: ./.github/actions/dependencies
+      - uses: SonarSource/sonarcloud-github-action@de2e56b42aa84d0b1c5b622644ac17e505c9a049
```

Comment:
why do you not use SonarSource/sonarcloud-github-action@master given in the example?
https://docs.sonarcloud.io/advanced-setup/ci-based-analysis/github-actions-for-sonarcloud/

Reply:
mh, was autogenerated by github, no idea, will change it

Reply:
https://github.com/SonarSource/sonarcloud-github-action

Reply:
could try setting sonarcloud as a separate job that runs after all the other linting checks (use 'needs:' in this case)

Reply:
used to be separate, didn't make the difference, I need it there because I need the coverage report